### PR TITLE
Multiclass object detection

### PR DESCRIPTION
**Pull request by sabina-xailient**
Added instruction to train multiclass object detection model

### CODE REVIEW

File: docs/multiclass_object_detector.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+* __augment_multiplier__: You can use this setting to augment your training data. 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+
+We recommend keeping all the other setting to default.
+
+### SSD
+
+Here is a sample configuration for the SSD architeture. 
+
+``` yml
+data:
```
Comment (jim-meyer) [inline docs/multiclass_object_detector.md:31]: I'm concerned about exposing config yml files to end users. The problem right now is that if there is a typo in the config.yml file, or some required value is missing, the training will simply fail and the user has no idea why. We (Xailient support) literally have to look at the training log files to figure out why the training failed; the user cannot figure this out on their own.
I've created https://app.asana.com/0/1162580331639025/1199992772008448/f to track this.

File: docs/multiclass_object_detector.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+debug: False
+```
+
+### YOLO
+
+Here is a sample configuration for the YOLO architeture. 
+
+``` yml
+data:
+  input_dim: 64
```
[the remark this answers is not in the document]
Comment (sabina-xailient) [reply docs/multiclass_object_detector.md:81]: I got this from the console_ml_code repo. I also confirmed that this is the correct config for yolo with Jim Zhang

File: docs/multiclass_object_detector.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+In Xailient Console, you can train two popular multi-class object detectors:
+
+1. [SSD](https://arxiv.org/abs/1512.02325)
+2. [YOLO](https://arxiv.org/abs/1804.02767)
+
+To train a multi-class object detector with SSD architecture, you can either use the default training setting or define your own.
+
+The training configuration file gives you full control of the hyperparameters for the trainng.
+
+There are the setting you can modify:
```
[the remark this answers is not in the document]
Comment (sabina-xailient) [reply docs/multiclass_object_detector.md]: fixed

File: docs/multiclass_object_detector.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  batch_size: 32
+  epochs: 200
+  augment_multiplier: 0 # 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+  classes: 'all'
+  encoding_iou_threshold: 0.5  # The IoU that will be used to match ground truth to priors when encoding
+  min_area: 0
+  # one of the aspect ratios has to be 1, the aspect ratios determine how many
+  # anchor boxes will be generated for each location on every prediction feature map
+  aspect_ratios: [1.0, 2.0, 0.5, 3.0, 0.3]  # The aspect ratios implemented in the original SSD paper.
+  l2_regularization: 0
```
[the remark this answers is not in the document]
Comment (sabina-xailient) [reply docs/multiclass_object_detector.md]: changed value of l2_regularization to 1.0e-3

File: docs/buildSdk.md [hunk 5]
```diff
@@ -6,7 +6,58 @@
...
+
+3. Choose your targeted platform and click <img src="../img/console/AI Models/BuildButton.png" height=30 width=50>.
+
+    <p align="center">
+    <img src="../img/console/AI Models/BuildSDK-ARM32.png" height=300>
+    </p>
+
+4. When the SDK build is completed, you will get an email notification, with an SDK download link.
+
+5. You can also download the SDK from the Console. Go to __AI Models__ page and locate the model for which you have build the SDK.
```
[the remark this answers is not in the document]
Comment (sabina-xailient) [reply docs/buildSdk.md]: fixed

File: docs/multiclass_object_detector.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sabina-xailient) [reply docs/multiclass_object_detector.md]: This was the default config for yolo in the console_ml_code repo. I checked with Jim Zhang regarding this and removed some values that were not required for training a Yolo model.

File: docs/selective_attention_network.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  batch_size: 64
+  epochs: 100
+  max_negative_sampling_ratio: 0.3  # Ratio of negative example sampling. Ratio = negative examples / positive examples
+  augment_multiplier: 0  # 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+
+test:
+  split_ratio: 0.1
+
+transforms:
+  Resize:
```
Comment (jim-meyer) [inline docs/selective_attention_network.md:74]: I included you on [a PR from Yousef](https://github.com/xailient/console_ml_code/pull/41) that will clean some of this up. So once that PR is merged you can update these docs based on that.

Comment (sabina-xailient) [reply docs/selective_attention_network.md]: Updated

**Review by jim-meyer — APPROVED** [review]
These changes look fine. But let's wait to merge this PR to master until we have https://app.asana.com/0/1162580331639025/1199992772008448/f fixed in production.

**Pull request merged**